src/sdgio-edit.owl

```diff
@@ -3293,6 +3307,45 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <ht
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020303> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020303> "2019-05-08T21:32:17Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.unep.org/sdg/SDGIO_00020303> "gender and youth")
+AnnotationAssertion(rdfs:comment <http://purl.unep.org/sdg/SDGIO_00020303> "Research conducted by CGIAR and its partners must be gender- sensitive and promote gender equity – that is, it is adapted to both the needs and the aspirations of poor women. The needs of young people must also be taken into account. Prioritizing rural and food sector entrepreneurship along agri-food supply chains will provide major opportunities for youth employment.")
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020303> "achievement of gender equity and youth employment"@en)
+ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020300> <http://purl.unep.org/sdg/SDGIO_00020303>)
+
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020304> (reformation of policies and institutions)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://hdl.handle.net/10947/3865>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.unep.org/sdg/SDGIO_00020304> "A cross-cutting objective which specifies the objective of the planned processes undertaken by the CGIAR such that they reform policies and institutions so that such policies and institutions effectively contributes to achieving the CGIAR system level objectives.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020304> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020304> "2019-05-09T00:36:14Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:comment <http://purl.unep.org/sdg/SDGIO_00020304> "Research must provide evidence as a basis for reforming agri-food policies and institutions to make them more conducive to pro-poor development, to improved nutrition and the sustainable management of natural resources. Again, this spans most CGIAR and partner research activities with a special need in economic and social sciences.")
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020304> "reformation of policies and institutions"@en)
+ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020300> <http://purl.unep.org/sdg/SDGIO_00020304>)
+
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020305> (increment of agri-food research productivity and of innovation in the agri-food system)
```

Comment:
The grammar here doesn't work.
Try: "Increment agri-food ... and innovation in ..."

Does the CGIAR have a reason for saying "increment" rather than "enhance" or "increase"?

Reply:
'increment' was part my rewording of the original "Capacity Development", which we found to be unacceptably vague. I changed it.